lib/workers/repository/update/branch/execute-post-upgrade-commands.ts

```diff
@@ -61,6 +66,37 @@ export async function postUpgradeCommandsExecutor(
         }
       }
 
+      let dataFilePath = '';
```

Comment:
```suggestion
      let dataFilePath: string | null = null;
```
The code base convention is to use `null` for representation of not set values when initializing.